InteractiveHtmlBom/generate_interactive_bom.py

```diff
@@ -236,6 +236,16 @@ def parse_edges(pcb):
                     bbox = d.GetBoundingBox()
                 else:
                     bbox.Merge(d.GetBoundingBox())
+    for m in pcb.GetModules():
+        for d in m.GraphicalItems():
+            if d.GetLayer() == pcbnew.Edge_Cuts:
+                parsed_drawing = parse_drawing(d)
```

Comment:
Build a combined list of drawings first, then run the parsing and bbox calculation code.

Copy-pasta is not a good pasta.